### DISCUSSION TITLE
Tab组件标头添加触摸滚动

### DISCUSSION THREAD

**Discussion by densen2014** [dotnetcore/BootstrapBlazor#4484] [Q&A]
添加  overflow-y: auto; 实现更好的滚动效果, 截图只是示意图. 实际没有滚动条
```
.tabs .of .tabs-nav-scroll {
    scrollbar-width: none;
    overflow-x: auto;
}
```

![image](https://github.com/user-attachments/assets/705b106c-c0a8-4aae-98dd-b93eeef8af47)

**Comment by ArgoZhang** [comment]
这个不是滚动的，PC 端都是禁止滚动的，因为有脚本控制需要完全显示
